bindings/nodejs/src/functions/i2c.cc

```diff
@@ -75,7 +77,11 @@ NAN_METHOD(bind_sol_i2c_set_slave_address)
     sol_i2c *i2c = (sol_i2c *)SolI2c::Resolve(jsI2c);
 
     int returnValue = sol_i2c_set_slave_address(i2c, info[1]->Uint32Value());
-    info.GetReturnValue().Set(Nan::New(returnValue));
+    if (returnValue < 0) {
+        info.GetReturnValue().Set(ReverseLookupConstant("E", abs(returnValue)));
```

Comment:
It is sufficient to use ```-returnValue``` here. No need for abs, since you already know that it's negative.